.github/workflows/linters.yml

```diff
@@ -0,0 +1,50 @@
+name: Linters check
+on: pull_request
+jobs:
+  linters_check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+            node-version: 12
+      - id: files
+        uses: jitterbit/get-changed-files@v1
+
+      - name: Install python/apt packages and create a report folder
+        run: |
+          echo "############ Installing apt packages ############"
+          sudo apt-get --no-install-recommends install -y build-essential curl python3-dev python3-pip python3-venv
+          echo "############ Installing python3 modules ############"
+          python3 -m venv .env
+          . .env/bin/activate
+          pip install -U pip wheel setuptools
+          pip install bandit
+          deactivate
+          mkdir -p linter_reports
+
+      - name: Python Bandit check
+        if: always()
```

Comment:
Why do we need the line?

Reply:
This will be necessary to run the check in case of success or failure of the previous checks, after the linter checks are combined into one workflow.